.github/workflows/gh-pages.yaml

```diff
@@ -25,8 +25,8 @@ jobs:
         with:
           mdbook-version: 'latest'
       
-      - name: Setup `mdbook-mermaid`
-        run: cargo install mdbook-mermaid
+      - name: Setup mdbook plugins (mermaid, linkcheck)
```

Comment:
Wonder whether this is enough to auto check in CI? If so then OK

Reply:
Yes, it checks automatically when `mdbook build` runs (which is the command ran just after: `mdbook build ./book`).